### PR TITLE
expose the last version time before the filter was generated

### LINKED ISSUE #13695: Pass timestamp along with the generated filter cascade
The blocklist should include a timestamp, so clients can determine whether the filter cascade can be relied upon for testing the blocklist state.

This timestamp should be synchronized with clock of the signing server, so that we do not inadvertently block add-ons that shouldn't be blocked, or allow add-ons that should have been blocked.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #13695 

@Rob--W the field is called `last_version_time` now because its not strictly connected to signing - though obviously a version can't be signed before it's been created.  (alternate names welcome)

### CODE REVIEW

**Comment by Rob--W** [comment]
`version` + `time` looks a bit stange. How about any of those:

- `generation_time`
- `creation_time`
- `last_modified_time`

**Comment by eviljeff** [comment]
> `version` + `time` looks a bit stange. How about any of those:
> 
>     * `generation_time`

https://github.com/mozilla/addons-server/pull/13749/commits/8e246af2b9d8d4bbcb4bbc3f84db2292def4e324

File: src/olympia/blocklist/cron.py [hunk 1]
```diff
@@ -21,12 +22,15 @@ def upload_mlbf_to_kinto():
         KINTO_BUCKET, KINTO_COLLECTION_MLBF, kinto_sign_off_needed=False)
     stats = {}
     key_format = get_mlbf_key_format()
+    generation_time = int(time.time() * 1000)
```
Comment (Rob--W) [inline src/olympia/blocklist/cron.py]: Is this temporary (until you have a better way of finding the time), or permanent?

In any case, a comment about the significance of this time stamp would be nice. Even if it is just a reference to #13695.

Comment (eviljeff) [reply src/olympia/blocklist/cron.py]: The conclusion from #13695 was the time on AMO was for all practical purposes the same time as on autograph so this is the best way, unless I'm misunderstanding the other's comments.  If you still disagree I'd comment in the issue.  

I'll add a comment in the code.

Comment (eviljeff) [reply src/olympia/blocklist/cron.py]: https://github.com/mozilla/addons-server/pull/13749/commits/c8379bbe5dd733744597b37e1446b80f7dd1b6cd

**Pull request merged**